lib/glific/jobs/minute_worker.ex

```diff
@@ -140,6 +144,9 @@ defmodule Glific.Jobs.MinuteWorker do
       "update_hsms" ->
         Partners.perform_all(&Templates.sync_hsms_from_bsp/1, nil, [])
 
+      "monthly_tasks" ->
+        Partners.perform_all(&Partners.send_dashboard_report/2, %{frequency: "MONTHLY"}, [])
+
```

Comment:
lets go with weekly one only for now and see how it is being used by NGOs and comment this one out for now